packages/Utils/src/index.tsx

```diff
@@ -1,15 +1,23 @@
-import classNames from "classnames";
 import React, { FunctionComponent } from "react";
-import ErrorMessage from "./ErrorMessage";
-import uniqueId from "./uniqueId";
 
 export default (ChildComponent: any) => {
   const WithUtils: FunctionComponent = (props: object): JSX.Element => {
-    const utils = {
-      ErrorMessage,
-      classNames,
-      uniqueId
-    };
+    const utils = new Proxy(
+      {},
+      {
+        get({}, libName: string) {
+          const allowedLibraries: string[] = [
+            "classNames",
+            "ErrorMessage",
+            "uniqueId",
+            "useDebounce"
+          ];
+          if (allowedLibraries.includes(libName)) {
+            return require(`./${libName}`).default;
```

Comment:
This is not a dynamic import no? would be dynamic import ``` import(`./${libName}`).default; ``` this will return a promise then you should await it, etc.

Reply:
yes, this should look something like this:

```
import('./${libname')
   .then(() => {
 .... some logic here
})
```

Reply:
> yes, this should look something like this:
> 
> ```
> import('./${libname')
>    .then(() => {
>  .... some logic here
> })
> ```

The require method does not return a promise.